src/node_env_var.cc

```diff
@@ -66,89 +66,73 @@ std::shared_ptr<KVStore> system_environment = std::make_shared<RealEnvStore>();
 Local<String> RealEnvStore::Get(Isolate* isolate,
                                 Local<String> property) const {
   Mutex::ScopedLock lock(per_process::env_var_mutex);
-#ifdef __POSIX__
+
   node::Utf8Value key(isolate, property);
-  const char* val = getenv(*key);
-  if (val) {
-    return String::NewFromUtf8(isolate, val, NewStringType::kNormal)
-        .ToLocalChecked();
-  }
-#else  // _WIN32
-  node::TwoByteValue key(isolate, property);
-  WCHAR buffer[32767];  // The maximum size allowed for environment variables.
-  SetLastError(ERROR_SUCCESS);
-  DWORD result = GetEnvironmentVariableW(
-      reinterpret_cast<WCHAR*>(*key), buffer, arraysize(buffer));
-  // If result >= sizeof buffer the buffer was too small. That should never
-  // happen. If result == 0 and result != ERROR_SUCCESS the variable was not
-  // found.
-  if ((result > 0 || GetLastError() == ERROR_SUCCESS) &&
-      result < arraysize(buffer)) {
-    const uint16_t* two_byte_buffer = reinterpret_cast<const uint16_t*>(buffer);
-    v8::MaybeLocal<String> rc = String::NewFromTwoByte(
-        isolate, two_byte_buffer, NewStringType::kNormal);
-    if (rc.IsEmpty()) {
-      isolate->ThrowException(ERR_STRING_TOO_LONG(isolate));
+  char* val = nullptr;
+  size_t initSz = 256;
+
+  // Allocate 256 bytes initially, if not enough reallocate.
+  val = Malloc(sizeof(char) * initSz);
+
+  int ret = uv_os_getenv(*key, val, &initSz);
+
+  if (UV_ENOBUFS == ret) {
+    // Buffer is not large enough, reallocate to the updated initSz
+    // and fetch env value again.
+    val = Realloc(val, sizeof(char) * initSz);
+
+    ret = uv_os_getenv(*key, val, &initSz);
+
+    // Still failed to fetch env value return emptry string.
+    if (UV_ENOBUFS == ret || UV_ENOENT == ret) {
       return Local<String>();
     }
-    return rc.ToLocalChecked();
+  } else if (UV_ENOENT == ret) {
+    return Local<String>();
   }
-#endif
-  return Local<String>();
+
+  Local<String> valueString =
+      String::NewFromUtf8(isolate, val, NewStringType::kNormal)
+          .ToLocalChecked();
+
+  if (nullptr != val) free(val);
+
+  return valueString;
 }
 
 void RealEnvStore::Set(Isolate* isolate,
                        Local<String> property,
                        Local<String> value) {
   Mutex::ScopedLock lock(per_process::env_var_mutex);
-#ifdef __POSIX__
+
   node::Utf8Value key(isolate, property);
   node::Utf8Value val(isolate, value);
-  setenv(*key, *val, 1);
-#else  // _WIN32
-  node::TwoByteValue key(isolate, property);
-  node::TwoByteValue val(isolate, value);
-  WCHAR* key_ptr = reinterpret_cast<WCHAR*>(*key);
-  // Environment variables that start with '=' are read-only.
-  if (key_ptr[0] != L'=') {
-    SetEnvironmentVariableW(key_ptr, reinterpret_cast<WCHAR*>(*val));
-  }
-#endif
+  uv_os_setenv(*key, *val);
+
```

Comment:
you left a blank line here, causing the linter to fail

Reply:
Hi devsnek, just removed the blank line and committed changes. 

Thank you.